pyscf/scf/dhf.py

```diff
@@ -206,16 +206,22 @@ def get_hcore(mol):
     h1e[n2c:,n2c:] = wn * (.25/c**2) - t
     return h1e
 
-def get_ovlp(mol):
+def get_ovlp(mol, offdiagonal = False):
     n2c = mol.nao_2c()
     n4c = n2c * 2
     c = lib.param.LIGHT_SPEED
 
     s = mol.intor_symmetric('int1e_ovlp_spinor')
     t = mol.intor_symmetric('int1e_spsp_spinor')
+    u = mol.intor_symmetric("int1e_sp_spinor")
```

Comment:
1. can be inside the `if offdiagonal` clause.
2. `get_ovlp(offdiagonal=True)` was not used anywhere. Is the offdiagonal necessary?

Reply:
1. Yes, sorry
2. Right. It will be needed for other parity-violating properties (also for energy in case of non-punctual nuclei). It will be used also in the pyECM code ([https://github.com/juanjoaucar/pyECM](https://github.com/juanjoaucar/pyECM))